src/pages/postgraphile/usage-cli.md

```diff
@@ -59,7 +59,7 @@ There are more CLI options available to customise the GraphQL server (these are
 - `--simple-collections [omit|both|only]`  
   "omit" (default) - relay connections only, "only" - simple collections only (no Relay connections), "both" - both
 - `--no-ignore-rbac`  
-  [RECOMMENDED] set this to exclude fields, queries and mutations that the user isn't permitted to access; this will be enabled by default in v5
+  [RECOMMENDED] set this to exclude fields, queries and mutations that the user account you connect to PostgreSQL with (from your connection string)  isn't permitted to access; this will be enabled by default in v5
```

Comment:
This line is auto-generated from the source, so will be lost; please make the change here instead:

https://github.com/graphile/postgraphile/blob/3d35995bce11526b6ab2754cd79c27af8ced16a0/src/postgraphile/cli.ts#L160